packages/web-components/src/components/card-group/card-group.ts

```diff
@@ -114,6 +114,11 @@ class DDSCardGroup extends StableSelectorMixin(LitElement) {
         this._childItemFooters.push(
           (e as HTMLElement).querySelector((this.constructor as typeof DDSCardGroup).selectorItemFooter)
         );
+        if (this.gridMode === 'border') {
+          e.setAttribute('border', '');
+        } else {
+          e.removeAttribute('border');
+        }
```

Comment:
```suggestion
        e.toggleAttribute('border', this.gridMode === 'border');
```

Reply:
thanks @kennylam! updated the styles and the toggleAttribute!